lib/appsignal.rb

```diff
@@ -267,15 +266,6 @@ def active?
       config && config.active? && extension_loaded?
     end
 
-    def is_ignored_error?(error)
-      Appsignal.config[:ignore_errors].include?(error.class.name)
-    end
-    alias :is_ignored_exception? :is_ignored_error?
-
-    def is_ignored_action?(action)
-      Appsignal.config[:ignore_actions].include?(action)
-    end
```

Comment:
Looking at this PR again I wonder if these methods are something we want to deprecate first before removing them? and then remove them in v3.0?